Player/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+
+namespace Player1
+{
+    interface IHasInfo
+    {
+        void ShowInfo();
+    }
+    interface IWeapon : IHasInfo
+    {
+        void Fire();
+        int Damage { get; }
+    }

```

Comment:
One file - one class/interface/enum

Reply:
and always so - only one there - 1 file per 1 class